src/components/calculatorUi.js

```diff
@@ -0,0 +1,77 @@
+import React,{useState} from 'react';
+import Button from './button';
+import calculate from '../logic/calculate';
+
+const Calculator = () => {
+  const btnFirst =['AC','+/-','%','/','7','8','9','*','4','5','6','-','1','2','3','+','0','.','=']
+  const [inputValue, setInputValue]=useState('')
+  const [numberOne,setNumberOne]=useState('')
+  const [numberTwo,setNumberTwo]=useState(0)
+  const [operator,setOperator]=useState('')
+
+
+  const handleNumber=(e)=>{
+    const value = e.target.textContent
+
+    if(numberOne !== ''){
+      if(value === '%' || value === '-' || value === '*' || value === '+' || value === '/' ){
+        console.log(numberOne)
+        setInputValue('')
+        setOperator(()=>value)
+        setInputValue(`${value}`)
+        return
+      }
+    }
+    if(!isNaN(value) && operator === ''){
+      setNumberOne(`${numberOne}${value}`)
+      setInputValue(`${inputValue}${value}`)
+    }else if(!isNaN(value) && operator !== ''){
+      if(!numberTwo){
+        setInputValue('')
+        setNumberTwo(`${value}`)
+        // setInputValue(`${inputValue}${value}`)
+      }
```

Comment:
- [ ] Kindly remove all commented-out codes from this file. Removing such codes make your work look neat 👍 💻